src/pages/help-centre/_article.tsx

```diff
@@ -9,6 +8,17 @@ import { SideTab, StyledLink } from 'components/elements'
 import { Container, SEO } from 'components/containers'
 import device from 'themes/device'
 
+export interface ArticleProps {
+    children?: React.ReactNode
+    header?: string
+    title?: string
+    description?: string
```

Comment:
We generally try to reach for `type` first unless there are special cases where we need an `interface`.